EIPS/eip-1.md

```diff
@@ -135,113 +134,87 @@ Random J. User
 
-If the EIP isn't ready, the editor will send it back to the author for revision, with specific instructions.
+If the EEP isn't ready, the editor will send it back to the author for revision, with specific instructions.
 
-Once the EIP is ready for the repository, the EIP editor will:
+Once the EEP is ready for the repository, the EEP editor will:
 
-- Assign an EIP number (generally the PR number or, if preferred by the author, the Issue # if there was discussion in the Issues section of this repository about this EIP)
+- Assign an EEP number (generally the PR number or, if preferred by the author, the Issue # if there was discussion in the Issues section of this repository about this EEP)
 
 - Merge the corresponding pull request
 
-- Send a message back to the EIP author with the next step.
+- Send a message back to the EEP author with the next step.
 
-Many EIPs are written and maintained by developers with write access to the Ethereum codebase. The EIP editors monitor EIP changes, and correct any structure, grammar, spelling, or markup mistakes we see.
+Many EEPs are written and maintained by developers with write access to the Ethereum codebase. The EEP editors monitor EEP changes, and correct any structure, grammar, spelling, or markup mistakes we see.
 
-The editors don't pass judgment on EIPs. We merely do the administrative & editorial part.
+The editors don't pass judgment on EEPs. We merely do the administrative & editorial part.
 
 ## History
 
-This document was derived heavily from [Bitcoin's BIP-0001] written by Amir Taaki which in turn was derived from [Python's PEP-0001]. In many places text was simply copied and modified. Although the PEP-0001 text was written by Barry Warsaw, Jeremy Hylton, and David Goodger, they are not responsible for its use in the Ethereum Improvement Process, and should not be bothered with technical questions specific to Ethereum or the EIP. Please direct all comments to the EIP editors.
-
-December 7, 2016: EIP 1 has been improved and will be placed as a PR.
-
-February 1, 2016: EIP 1 has added editors, made draft improvements to process, and has merged with Master stream.
-
-March 21, 2018: Minor edits to accommodate the new automatically-generated EIP directory on [eips.ethereum.org](http://eips.ethereum.org/).
-
-May 29, 2018: A last call process was added.
-
-See [the revision history for further details](https://github.com/ethereum/EIPs/commits/master/EIPS/eip-1.md), which is also available by clicking on the History button in the top right of the EIP.
-
-### Bibliography
-
-[EIP5]: https://github.com/ethereum/EIPs/blob/master/EIPS/eip-5.md
-[EIP101]: https://github.com/ethereum/EIPs/issues/28
-[EIP90]: https://github.com/ethereum/EIPs/issues/90
-[EIP86]: https://github.com/ethereum/EIPs/issues/86#issue-145324865
-[devp2p]: https://github.com/ethereum/wiki/wiki/%C3%90%CE%9EVp2p-Wire-Protocol
```

Comment:
If there is some links about P2P protocol of EOS, we could update this one to `[P2P]` and point there.